spec/models/blip-spec.js

```diff
@@ -67,6 +67,30 @@ describe('Blip', function () {
     expect(blip.isNew()).toBe(false)
   })
 
+  it('status is new', function () {
```

Comment:
Can you add topic and description to these tests so it's easier to follow which params are which?

Reply:
Sorted.